server/readme.md

```diff
@@ -1,12 +1,99 @@
 # U.S. Forest Service Intake Module Server
 
-For development: `npm start` and visit http://localhost:8080
+A running Postgresql database is required in order to run the server locally.  Please make sure you have installed [Postgresql](https://www.postgresql.org/) locally and created a database for this project.
+
+Then run `yarn install` to install dependencies.
+
+## Environment variables
+
+There are a number of environment variables that are required to be set in order to run tests
```

Comment:
Do you mind stubbing out the db/ s3 and middlelayer service for your tests so that the env vars do not need to be provided for testing?